microsetta_interface/templates/account_details.jinja2

```diff
@@ -68,6 +68,32 @@
                     $("#state").empty();
                 }
 
+                // Special case to prevent users outside Japan from using Japanese as their language
```

Comment:
why?

Reply:
We don't have the resources to translate all future changes into Japanese in the future, so we're limiting that locale to Japanese residents in an effort to allow THDMI Japan participants to view their results, while preventing anyone in the US from using the language. We may revisit this decision in the future.

Reply:
if the current content works and is correct, why not defer making this special case until a point of deviation in the future?

Reply:
This has been removed from the PR